### DISCUSSION TITLE
provide metric for custom rule instead of alerting

### DISCUSSION THREAD

**Discussion by y0zg** [jertel/elastalert2#1270] [Ideas]
I'd like to run some rule(e.g. find ratio of INFO/ERROR log levels) and instead of running alerting, provide metric which will be scraped by prometheus and further processed. Is is possible to achieve this based on current functionality?

Thanks, 
OK

**Comment by jertel** [comment]
Prometheus metrics only include counts of various statistics, such as number of alerts sent. ElastAlert 2 supports command line alerts, which let you invoke a shell script, so perhaps you could use that type of alert to push your metric somewhere.
